### PR TITLE
Bogdan negura connect

### PR DESCRIPTION
**Pull request by BogdanNegura**
Please Do Not Merge

### CODE REVIEW

File: web/components/connect/connect.css [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+}
+
+.avatar {
+  display: flex;
+  justify-content: center;
+  min-width: 302px;
+  height: 302px;
+}
+
+.avatar img {
```
Comment (marianzburlea) [inline web/components/connect/connect.css]: refactor using BEM

File: web/components/connect/connect.css [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  list-style: none;
+  width: 320px;
+  padding: 10px;
+  margin: 25px;
+}
+
+.svg-files img {
```
Comment (marianzburlea) [inline web/components/connect/connect.css]: refactor using BEM

File: web/components/connect/connect.css [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  border-radius: 50%;
+  margin-top: 30px;
+  padding: 15px;
+  margin: 3px;
+}
+li:nth-child(1) {
```
Comment (marianzburlea) [inline web/components/connect/connect.css]: the selector is too generic
refactor using BEM

File: web/components/connect/connect.css (hunk 2 above)
Comment (BogdanNegura) [reply web/components/connect/connect.css]: I get it done

**Pull request merged**